validation/src/main/scala/hmda/validation/rules/lar/validity/V485.scala

```diff
@@ -0,0 +1,31 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.{ Predicate, Result }
+import hmda.validation.rules.EditCheck
+import scala.util.Try
+
+object V485 extends EditCheck[LoanApplicationRegister] {
+
+  def apply(lar: LoanApplicationRegister): Result = {
+    val applicant = lar.applicant
+    when(applicant.coRace1 is containedIn(1 to 5)) {
+      (applicant.coRace2 is validRace) and
+        (applicant.coRace3 is validRace) and
+        (applicant.coRace4 is validRace) and
+        (applicant.coRace5 is validRace)
+    }
+  }
+
+  def validRace: Predicate[String] = new Predicate[String] {
```

Comment:
I see how this makes the language for this edit a little nicer, but I don't think it belongs here. It leaks implementation of the language into its usage. This also breaks the typeclass pattern, which we are using for the DSL (there is a refactor that I am working on about this topic that will explain this in much further detail). If you want to keep it this way, I suggest putting this function in [HmdaDsl](https://github.com/cfpb/hmda-platform/blob/master/validation/src/main/scala/hmda/validation/dsl/HmdaDsl.scala) since this language is pretty specific to HMDA (and possibly including `blankOr1to5` as an inner function to `validRace`)